### ISSUE TITLE
Exception thrown on android

### ISSUE THREAD

**Issue by lyubo** [Kerumen/react-native-awesome-card-io#32]
On Android platform the following exception is thrown:
```
04-07 14:56:53.898  3185  4686 E unknown:React: java.lang.IllegalArgumentException: Can only use lower 16 bits for requestCode
04-07 14:56:53.898  3185  4686 E unknown:React: 	at android.support.v4.app.BaseFragmentActivityGingerbread.checkForValidRequestCode(BaseFragmentActivityGingerbread.java:88)
04-07 14:56:53.898  3185  4686 E unknown:React: 	at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:856)
04-07 14:56:53.898  3185  4686 E unknown:React: 	at com.cardio.RNCardIOModule.scanCard(RNCardIOModule.java:44)

```

This is due to the following constant:
`  public static final int CARD_IO_SCAN = 49463195;`
used here:
`activity.startActivityForResult(scanIntent, CARD_IO_SCAN);
`
It should be set to something less than 0xFFFF. According to card-io documentation this is an arbitrary value. Setting this constant to 1 works fine.

**Comment by Kerumen** [comment]
Can you submit a PR for this? I would happy to merge it!

**Comment by lyubo** [comment]
PR created.

**Comment by Kerumen** [comment]
Fixed by 0bdafd6cd23ce1e672c2f5202fef02ab0c33d741.
